test/unittests/libethereum/GasPricer.cpp

```diff
@@ -119,8 +119,8 @@ BOOST_AUTO_TEST_CASE(basicGasPricer_bcUncleTest_Frontier)
 
 BOOST_AUTO_TEST_CASE(basicGasPricer_bcUncleTest_Homestead)
 {
-	u256 _expectedAsk = 155633980282;
-	u256 _expectedBid = 1;
+    u256 _expectedAsk = 155632494086;
```

Comment:
Why did these change?

Reply:
If it's because these tests use NoProof now, I'm not sure, why does it affect gas pricing?

Reply:
I don't know.

Reply:
These tests seem to check something that is not used anywhere, so it's fine for now, we will probably delete them later.

But I think this change was needed because NoProof didn't calculate block gas limit, so you might need to roll this back after rebase on #5299